src/System.Device.Gpio/Interop/Unix/libgpiod/V1/SafeLineHandle.cs

```diff
@@ -7,31 +7,53 @@
 namespace System.Device.Gpio.Libgpiod.V1;
 
 /// <summary>
-/// Pointer to a pin.
+/// Pointer to a pin (Not a real SafeLineHandle, because we need to align its finalization with the owning object)
 /// </summary>
-internal class SafeLineHandle : SafeHandle
+internal sealed class SafeLineHandle : IDisposable
```

Comment:
NIT: Given this is no longer an actual SafeHandle, we should likely rename this to something like `LineHandle` or something.